style.css

```diff
@@ -0,0 +1,64 @@
+/* * {
```

Comment:
I would like to advise you to consider importing every kind of font used in the project from [google fonts](https://fonts.google.com/) so can work properly on every machine.